packages/types/src/organizationSuggestion.ts

```diff
@@ -0,0 +1,19 @@
+import type { ClerkResource } from './resource';
+
+export type OrganizationSuggestionStatus = 'pending' | 'accepted';
```

Comment:
Will this in the feature include 'revoked' or 'rejected' ?

I am trying to figure out why we cannot reuse `OrganizationInvitationStatus`

Reply:
Yes we are planning to introduce more statuses in the future, so it's better to create an explicit type instead